build.gradle

```diff
@@ -115,3 +116,11 @@ task wrapper(type: Wrapper) {
     gradleVersion = '2.11'
     distributionUrl = "http://services.gradle.org/distributions/gradle-${gradleVersion}-all.zip"
 }
+
+if (JavaVersion.current().isJava8Compatible()) {
+    allprojects {
+        tasks.withType(Javadoc) {
+            options.addStringOption('Xdoclint:none', '-quiet')
+        }
+    }
+}
```

Comment:
I am not sure what this does. Please open a separate PR for that, if this change was intentional.